index.html

```diff
@@ -174,95 +182,98 @@ <h3>
         <article class="languages_container">
           <div class="lang tags">
             <h3>Languages</h3>
-            <img class="downBTN" src="downBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/downBTN.png" alt="togbtn" />
           </div>
           <ul class="toggle">
             <li class="lan">
-              <img src="js.png" alt="js icon" />
+              <img src="images/js.png" alt="js icon" />
               <p>JavaScript</p>
             </li>
             <li class="lan">
-              <img src="html.png" alt="html icon" />
+              <img src="images/html.png" alt="html icon" />
               <p>HTML</p>
             </li>
             <li class="lan">
-              <img src="css.png" alt="css icon" />
+              <img src="images/css.png" alt="css icon" />
               <p>CSS</p>
             </li>
           </ul>
           <div class="lang">
             <h3>Frameworks</h3>
-            <img class="downBTN" src="rightBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/rightBTN.png" alt="togbtn" />
           </div>
           <div class="lang">
             <h3>Skills</h3>
-            <img class="downBTN" src="rightBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/rightBTN.png" alt="togbtn" />
           </div>
         </article>
       </section>
       <section class="contactForm">
         <div class="txt">
           <h2>Contact me</h2>
           <p>
-            If you have an application you are interested in developing, a feature
-            that you need built or a project that needs coding. I’d love to help
-            with it!
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
           </p>
         </div>
         <form action="https://formspree.io/f/mknyobde" method="post">
           <input
             type="text"
             name="name"
             id="name"
-            minlength="30"
-            placeholder="Lorem Ipsum"
+            max-length="30"
+            placeholder="Your name (first and last)"
             required
           />
           <input
             type="email"
             name="email"
             id="email"
-            placeholder="loremipsum@gmail.com"
+            placeholder="your email address"
             required
           />
-          <textarea name="message" id="message" cols="30" rows="10" required>Write your message here</textarea
-          >
-          <input type="submit" value="Get In Touch" />
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            placeholder="Write your message here..."
+            maxlength="500"
+            required
+          ></textarea>
+          <input type="submit" value="Get in touch" />
```

Comment:
- [ ] 👍 Great job on implementing styles for the default and hover stages of the button! However, please ensure that you also include styles for the **pressed stage** to complete the button's visual behavior as specified in the design. By adding the active stage styles, you will ensure a consistent and interactive user experience. Keep up the good work! 💪🎉

![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/55840999/c2cbb97d-45d5-46a5-a8d1-0c75f7b0fee1)